anvil/anvil-compiler/src/main/java/com/duckduckgo/anvil/compiler/ContributeToActivityStarterCodeGenerator.kt

```diff
@@ -111,6 +122,39 @@ class ContributeToActivityStarterCodeGenerator : CodeGenerator {
                     )
                     .build(),
             )
+            .addFunction(
```

Comment:
I think we should skip adding this function if the screenName was blank

```kotlin
.apply {
  if (screenName.isNotBlank()) {
    addFunction(...)
  }
}

Reply:
Oh that's cool indeed! So depending on `screenName` value, the implementation can be different, and use that to just provide an empty mapper if no `screenName` provided. Makes sense.